assets/js/analyzer.js

```diff
@@ -143,7 +143,7 @@ function populateSecondaryAnalyzerList() {
 }
 
 function analyze() {
-    if($('#primaryAnalyzerMode').val() === null) {
+    if($('#primaryAnalyzerMode').val() === null || $('#morphAnalyzerInput').val() === '') {
```

Comment:
I agree. Idea: make the analyzer input required and before executing generate or analyze, do a form validity check.